compiler/src/dotty/tools/dotc/sbt/ExtractAPI.scala

```diff
@@ -597,11 +660,21 @@ private class ExtractAPICollector(using Context) extends ThunkHolder {
       // FIXME: The API of a class we send to Zinc includes the signatures of
```

Comment:
I think we can just completely remove this FIXME comment, the inlineBodyCache solution seems good enough to me.

Reply:
done :)